Source/Mosa.Kernel.x86/IDT.cs

```diff
@@ -2146,9 +2147,10 @@ private static void IRQ255()
 	/// Interrupts the handler.
 	/// </summary>
 	/// <param name="stackStatePointer">The stack state pointer.</param>
-	private static unsafe void ProcessInterrupt(uint stackStatePointer)
+	[Plug("Mosa.Runtime.Interrupt::Process")]
+	private static unsafe void ProcessInterrupt(Pointer stackStatePointer)
 	{
-		var stack = (IDTStack*)stackStatePointer;
+		var stack = (IDTStack*)stackStatePointer.ToInt32();
```

Comment:
I think we should use ``ToUInt32()`` here since an address can't be negative.

Reply:
Changed.